### DISCUSSION TITLE
Setting Up Web Relay For The First Time

### DISCUSSION THREAD

**Discussion by mrbrwnstne** [Ylianst/MeshCentral#5743] [Q&A]
Good morning, everyone,

This is my first time delving into something like this, so my knowledge is quite limited, and I've encountered a bit of a challenge. I have Mesh Central running on a cloud server (Linode), and I can connect to my home PC without any issues. However, I'm facing difficulties implementing the web relay. I've added the relayPort to the config.json, but I can't seem to view anything, such as my router connected through my home PC. I'm sure it's something very simple. Any help would be greatly appreciated.

Thank you.

**Comment by si458** [comment]
for starters make sure the relay port is open in your firewall as when you click the HTTP/HTTPS button, youe browser will open to https://MC.MYDOMAIN.COM:RELAYPORT
also some times you might need to click HTTPS instead of HTTP

i have also experience where the web relay DOESNT work with BT home hubs for example,
if i create a relay group, select a computer at a site as the device, then add the router into the relay group (using its ip),
then click HTTP or HTTPS, i just get a white screen,

but then again i can use the web relay with a linux machine which has the agent installed and running openmediavault no problem

sadly the web relay is helpful for things but not everything

one other thing you could also try is using the `relayDNS` option instead, and assign a DNS name to be used instead of a port
just make sure to add the relaydns to name in the letsencrypt section if you use it
